docs/usage/setup.md

```diff
@@ -88,7 +88,7 @@ You can download the latest build from our Google Cloud Storage (GCS) bucket (co
 SYSTEM=ubuntu-x86_64 # or macos-x86_64 or macos-arm64
 curl https://storage.googleapis.com/mysten-walrus-binaries/walrus-latest-$SYSTEM -o walrus
 chmod +x walrus
-mv walrus ~/.local/bin
+mv walrus ~/.local/bin/
```

Comment:
Thanks, this prevents issues if the directory does not exist yet. Maybe we can make this even better by first creating the directory?
```suggestion
mkdir -p ~/.local/bin
mv walrus ~/.local/bin/
```

Reply:
The actual solution of me is 
```
mv walrus /usr/local/bin/
```

Reply:
I think it is a default path to mv binary in both Linux and Mac. Is it better to mv walrus to that path?

Reply:
In the end it's up to the user to put it into `/usr/bin`, `/usr/local/bin`, `~/bin`, `~/.local/bin` (all of which are somewhat standard directories), or any other directory that is included in the `PATH` environment variable. To me, `~/.local/bin` or `/usr/local/bin` make the most sense. 

Maybe we should change this to `/usr/local/bin`, which AFAIK exists on macOS and all Linux distros. I will create a separate PR for this.